test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -36,11 +36,39 @@ public void elementsAreUnique() throws Exception {
         assertNotUnique(null, "a", "b", null);
     }
 
+    @Test
+    public void noElementsAreNull() throws Exception {
```

Comment:
The method name doesn't adhere to all the test cases inside. It depicts: no Elements being null but test cases include "assertAreNull". Please refer to the discussion we had in the tutorial regarding the 3-part naming and how to generalize into 1-part.